torch/_torch_docs.py

```diff
@@ -3824,6 +3824,13 @@ def merge_dicts(*dicts):
 (see :func:`torch.squeeze`), resulting in both the :attr:`values` and
 :attr:`indices` tensors having 1 fewer dimension than the :attr:`input` tensor.
 
+.. warning::
```

Comment:
Let's turn this warning into a note and simplify it to:

"When there are multiple valid kth values this function may nondeterministically return indices for any of them."